### DISCUSSION TITLE
关于4k汉化

### DISCUSSION THREAD

**Discussion by MillerTheBest** [GpointChen/FFXIVChnTextPatch-GP#97] [Q&A] [answered]
我用的是2月9日的release。有两个问题：
1. 请问应该用4k文件夹中的font替换汉化包中的font，还是把font-4k放在原本font旁边？
2. 汉化的时候选的是简体中文，可是汉化之后仍然是繁体，请问是哪里操作错误了么？
谢谢

**Comment by GpointChen** [comment, marked as answer]
1. 替換
2. 簡體中文選項只適用於「使用中國服檔案漢化」，代表不做簡繁轉換而已。如果使用CSV漢化請額外下載簡體CSV。
